2020-04-08 18:12:14,256 ERROR: Exception on /reset_pasword_request [POST] [in /home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py: 1891]
Traceback (most recent call last):
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/nurbek/Desktop/microblog/app/routes.py", line 165, in reset_password_request
    send_password_reset_email(user)
  File "/home/nurbek/Desktop/microblog/app/email.py", line 19, in send_password_reset_email
    user=user, token=token),
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/loaders.py", line 129, in load
    code = environment.compile(source, name, filename)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/nurbek/Desktop/microblog/app/templates/email/reset_password.txt", line 7, in template
    {{ url_for('reset_password' token=token, _external=True) }}
jinja2.exceptions.TemplateSyntaxError: expected token ',', got 'token'
2020-04-08 18:13:49,599 INFO: Microblog [in /home/nurbek/Desktop/microblog/app/__init__.py: 50]
2020-04-08 18:13:59,928 ERROR: Exception on /reset_pasword_request [POST] [in /home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py: 1891]
Traceback (most recent call last):
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/nurbek/Desktop/microblog/app/routes.py", line 163, in reset_password_request
    send_password_reset_email(user)
  File "/home/nurbek/Desktop/microblog/app/email.py", line 19, in send_password_reset_email
    user=user, token=token),
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/loaders.py", line 129, in load
    code = environment.compile(source, name, filename)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/nurbek/Desktop/microblog/env/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/nurbek/Desktop/microblog/app/templates/email/reset_password.txt", line 7, in template
    {{ url_for('reset_password' token=token, _external=True) }}
jinja2.exceptions.TemplateSyntaxError: expected token ',', got 'token'
2020-04-08 18:15:08,170 INFO: Microblog [in /home/nurbek/Desktop/microblog/app/__init__.py: 50]
2020-04-08 19:48:09,491 INFO: Microblog [in /home/nurbek/Desktop/microblog/app/__init__.py: 50]
2020-04-08 19:49:03,008 INFO: Microblog [in /home/nurbek/Desktop/microblog/app/__init__.py: 50]
2020-04-08 20:57:00,908 INFO: Microblog [in /home/nurbek/Desktop/microblog/app/__init__.py: 51]
2020-04-08 21:00:14,376 INFO: Microblog [in /home/nurbek/Desktop/microblog/app/__init__.py: 51]
2020-04-08 21:01:34,620 INFO: Microblog [in /home/nurbek/Desktop/microblog/app/__init__.py: 51]
2020-04-08 21:03:15,591 INFO: Microblog [in /home/nurbek/Desktop/microblog/app/__init__.py: 51]
2020-04-08 21:05:16,094 INFO: Microblog [in /home/nurbek/Desktop/microblog/app/__init__.py: 51]
2020-04-09 18:23:40,708 INFO: Microblog [in /home/nurbek/Desktop/microblog/app/__init__.py: 51]
